tutorials/4 Plugin Developer/Create new QRenderer.ipynb

```diff
@@ -276,26 +301,30 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# The entire QDesign uses three tables. \n",
     "a_skeleton.write_qgeometry_table_names_to_file('./simple_output.txt')"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here another example where we only export the shapes related to one RouteMeander class instance `cpw1`. Note that the text file will only contains a single table in this case, because the RouteMeander only draws using paths, and not junctions or poly."
```

Comment:
Suggest:
Note that the text file will only contain a single name for table.  In this case, because the RouteMeander only draws using paths, and not junctions or poly."